client/state/site-settings/exporter/selectors.js

```diff
@@ -1,5 +1,127 @@
+/**
+ * Internal dependencies
+ */
+import i18n from 'lib/mixins/i18n';
+
+import moment from 'moment';
+
 import { States } from './constants.js';
 
+function mapOptions( state, section, name, mapFunc ) {
+	const rawData = state.siteSettings.exporter.data.get( 'advancedSettings' );
+	if ( !rawData || rawData.count() === 0 ) return [];
+
+	return rawData.get( section ).get( name ).map( mapFunc );
+}
+
+export function getAuthorOptions( state, section ) {
+	return mapOptions( state, section, 'authors', ( author ) => ( {
+		value: author.get( 'ID' ), label: author.get( 'name' )
```

Comment:
would you mind putting these on two lines? I'm usually all for merging on a single line when we are simply passing the variables, but it makes me uneasy seeing this with the function calls